walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -20,47 +20,19 @@
 
 public final class FindMeetingQuery {
 
-  // Add a time range to Collection results
-  private void addRangeToResults(int validRangeStart, int validRangeEnd,
-      boolean inclusive, Collection<TimeRange> results, long duration) {
+  // Add a time range to Collection validRanges
+  private void addRange(int validRangeStart, int validRangeEnd,
+      boolean inclusive, Collection<TimeRange> validRanges, long duration) {
     TimeRange validRange = TimeRange.fromStartEnd(validRangeStart, validRangeEnd, inclusive);
     if (validRange.duration() >= duration) {
-      results.add(validRange);
+      validRanges.add(validRange);
     }
   }
 
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    Collection<TimeRange> results = new ArrayList<TimeRange>();
-    Collection<String> attendees = request.getAttendees();
-    long duration = request.getDuration();
-
-    // Not possible to have an event longer than a full day
-    if (duration > TimeRange.WHOLE_DAY.duration()) {
-      return results;
-    }
-
-    // If no one's attending, there are no conflicts
-    if (attendees.isEmpty()) {
-      results.add(TimeRange.WHOLE_DAY);
-      return results;
-    }
-
-    ArrayList<TimeRange> invalidRanges = new ArrayList<TimeRange>();
-    for (Event event : events) {
-      // If there are attendees in common between the request and this event
-      if (!Collections.disjoint(attendees, event.getAttendees())) {
-        // Requested meeting can't be at same time as this event
-        invalidRanges.add(event.getWhen());
-      }
-    }
-
-    if (invalidRanges.isEmpty()) {
-      results.add(TimeRange.WHOLE_DAY);
-      return results;
-    }
-
-    Collections.sort(invalidRanges, TimeRange.ORDER_BY_START);
-    
+  // Return time ranges that don't conflict with invalidRanges
```

Comment:
I think this could be a little bit clearer. In particular, what role does duration play in determining what time ranges get returned?